### DISCUSSION TITLE
Errors

### DISCUSSION THREAD

**Discussion by korich78** [XTLS/Xray-core#3382] [General]
Hello, in general, I updated Xray to 1.8.12 and v2rayN to 6.43, which ultimately caused errors; when I start the key, it throws out the following errors: “Exception 0xc0000005 0x8 0x0 0x0
PC=0x0
runtime.asmstdcall(0x20)
runtime/sys_windows_386.s:44 +0x2a fp=0x12fe28 sp=0x12fe24 pc=0x47b70a
eax 0x8
ebx 0x1bdc350
ecx 0x0
edx 0x12fe58
edi 0x12fe2c
esi 0x12fe8c
ebp 0x12fe2c
esp 0x12fe20
eip 0x0
eflags 0x10206
cs 0x1b
fs 0x3b
gs 0x0"

System: Windows 7 32-bit maximum, installed updates that are necessary for normal operation + Microsoft Network + Visual

**Comment by uuonda** [comment]
Latest Xray version you can officially run on Windows 7 is v1.8.6. It's no longer supported.

**Reply by korich78** [reply]
And here now a problem arises, my keys either work normally or die, that is, one specific key has a delay of 120 and a speed of -1, and so on, unfortunately, all the time. Yes, you can of course blame the problem on the server, but they work on my phone without any problems

**Reply by uuonda** [reply]
But what does it have to do with Xray?

**Reply by korich78** [reply]
I am not competent in this matter, but the suspicion fell on the fact that the Xray kernel is not updated, and the keys still live so-so

**Reply by uuonda** [reply]
Xray is regularly updated. It's your operating system that is not. Anyway I don't think v1.8.6 client is incompatible with the latest Xray server.

**Comment by ErbiGuard** [comment]
"If you don't want to revert to version v1.8.6, you can compile it yourself according to the tutorial in this discussion, at your own risk. https://github.com/XTLS/Xray-core/discussions/3373"
